app/src/marketplace/i18n/en.po

```diff
@@ -1330,6 +1330,9 @@ msgstr "Subscribers"
 msgid "productPage##stats##adminFee"
 msgstr "Admin Fee"
```

Comment:
If it's "instead" then shouldn't we drop translations for the fee? Or is it used someplace else?

Reply:
It's still used when the Admin is creating the product. For the public facing product page, the perspective is flipped - they care more about what they're getting, rather than how much the admin is making, so we still need both.

Reply:
Actually with the newest editor version setting the admin fee has a different translation key so we can drop this `adminFee` under `stats` namespace.